contrib/plugins/helloworld.py

```diff
@@ -19,7 +19,7 @@ def hello(plugin, name="world"):
     return s
 
 
-@plugin.method("init")
+@plugin.init()
```

Comment:
This is actually a breaking change for existing plugin libs (i.e. mine :smile: ) 

Is that sufficient to require a version bump of pylightning ?

Reply:
Well, plugins aren't released yet! Which is why I washed to get this right before they are...